e2e/cypress/tests/integration/keyboard_shortcuts/dot_menu_spec.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @keyboard_shortcuts
+import {beUnread} from '../../support/assertions';
```

Comment:
Just me being nitpicky :)
```suggestion
// Group: @keyboard_shortcuts

import {beUnread} from '../../support/assertions';
```